.github/workflows/ci.yml

```
@@ -9,7 +9,9 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v1
```

lets use `actions/setup-go@v2.1.3`
```suggestion
        uses: actions/setup-go@v2.1.3
```